README.md

```diff
@@ -480,6 +485,64 @@ Linking.getInitialURL().then((url) => {
 });
 ```
 
+## In-App Forms
+
+[In-app forms](https://help.klaviyo.com/hc/en-us/articles/34567685177883) are messages displayed to mobile app users while they are actively using an app. You can create new in-app forms in a drag-and-drop editor in the Sign-Up Forms tab in Klaviyo. Follow the instructions in this section to integrate forms with your app. The SDK will display forms according to their targeting and behavior settings and collect delivery and engagement analytics automatically.
+
+### Prerequisites
+
+- Using Version 1.2.0 and higher
+- Import the Klaviyo module
+
+#### Android
+
+In your Application class you **must** call `Klaviyo.registerForLifecycleCallbacks(applicationContext)`, due to timing issues, this cannot be achieved from the RN layer at this time.
+
+```kotlin
+// Application subclass
+import android.app.Application
+import com.klaviyo.analytics.Klaviyo
+class YourApplication : Application() {
+    override fun onCreate() {
+        super.onCreate()
+        /* ... */
+
+        // Initialize is required before invoking any other Klaviyo SDK functionality
+        Klaviyo.initialize("KLAVIYO_PUBLIC_API_KEY", applicationContext)
+
+        // If unable to call initialize, you must at least register lifecycle listeners:
+        Klaviyo.registerForLifecycleCallbacks(applicationContext)
+    }
+}
+```
+
```

Comment:
```suggestion
```

With https://github.com/klaviyo/klaviyo-react-native-sdk/pull/207 we'll be able to exclude this section from the README.